### PR TITLE
CB-12148 added null check for availability zones in aws validateRegionAndZone

### PR DESCRIPTION
**Pull request by afarsang**
See detailed description in the commit message.

### CODE REVIEW

File: cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java [hunk 1]
```diff
@@ -111,6 +111,10 @@ public void prerequisites(AuthenticatedContext ac, CloudStack stack, Persistence
     private void validateRegionAndZone(CloudCredential cloudCredential, Location location) {
         CloudRegions regions = awsPlatformResources.regions(cloudCredential, location.getRegion(), Collections.emptyMap(), true);
         List<AvailabilityZone> availabilityZones = regions.getCloudRegions().get(location.getRegion());
+        if (availabilityZones == null) {
```
Comment (bergerdenes) [inline cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java:114]: or empty

**Review by bergerdenes — CHANGES_REQUESTED** [review]
please fix check

File: cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java (hunk 1 above)
Comment (biharitomi) [reply cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java:114]: Collections.isEmpty() ? 😸

Comment (afarsang) [reply cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsSetup.java:114]: actually the empty is handled in the next lines nicely

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**